pkgs/development/python-modules/schemainspect/sql_import.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/pyproject.toml b/pyproject.toml
+index 3fe0463..cc2df64 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -4,11 +4,13 @@ version = "3.1"
+ authors = ["Robert Lechte <robertlechte@gmail.com>"]
+ license = "Unlicense"
+ readme = "README.md"
++include = [{ path = "schemainspect/pg/sql", format = ["sdist", "wheel"] }]
+ description = "Schema inspection for PostgreSQL (and possibly others)"
+
+ repository = "https://github.com/djrobstep/schemainspect"
+ homepage = "https://github.com/djrobstep/schemainspect"
+
++
+ [tool.poetry.dependencies]
+ python = ">=3.7,<4"
+ sqlalchemy = "*"
```

Comment:
Can be removed, or the PEP-517 fix could be moved here. Please submit any patches as upstream PRs and fetch them with `fetchPatch`

Reply:
Okay, I've submitted the patch in https://github.com/djrobstep/schemainspect/pull/87